engine/engine/src/engine.cpp

```diff
@@ -17,6 +17,7 @@
 #include "engine_private.h"
 
 #include <dmsdk/dlib/vmath.h>
+
 #include <sys/stat.h>
```

Comment:
Should this still be here?

Reply:
I have other changes in the private repo as well for this case. So it's a two-step process.